### ISSUE TITLE
CUDA problem when following the `get started`

### ISSUE THREAD

**Issue by ironheads** [nex-mpi/nex-code#42]
when follow the steps
```bash
conda env create -f environment.yml
./download_demo_data.sh
conda activate nex
python train.py -scene data/crest_demo -model_dir crest -http
```
Here comes the problem, I don't know what happens. 
``` 
"train.py" in <module>
  751:  train()
  train.py
"train.py" in train
  633:  output = model(dataset.sfm, feature, output_shape, sel)
  train.py
"module.py" in _call_impl
  889:  result = self.forward(*input, **kwargs)
"train.py" in forward
  334:  warp, ref_coords = computeHomoWarp(sfm,
  train.py
"train.py" in computeHomoWarp
  158:  prod = coords @ pt.transpose(Hs, 1, 2).cuda()
  train.py
RuntimeError: CUDA error: CUBLAS_STATUS_INTERNAL_ERROR when calling `cublasCreate(handle)`
```

**Comment by pureexe** [comment]
Here is how to solve this problem
https://discuss.pytorch.org/t/cuda-error-cublas-status-internal-error-when-calling-cublascreate-handle/114341/11

**TL;DR. re-install pytorch with CUDA11.1 and CUDA11.x**

**Comment by ironheads** [comment]
That really works. Thanks a lot.

> Here is how to solve this problem https://discuss.pytorch.org/t/cuda-error-cublas-status-internal-error-when-calling-cublascreate-handle/114341/11
> 
> **TL;DR. re-install pytorch with CUDA11.1 and CUDA11.x**

That really works. Thanks a lot!!!!!!
